mkdocs/__main__.py

```diff
@@ -91,6 +91,7 @@ def __init__(self, log_name='mkdocs', level=logging.INFO):
 remote_name_help = ("The remote name to commit to for Github Pages. This "
                     "overrides the value specified in config")
 force_help = "Force the push to the repository."
+no_history_help = "Force new commit without parent history."
```

Comment:
Uhh so I can easily guess what this description means, but still not fully sure. Also maybe the severity of this action is underrepresented.

So maybe something like this?

```suggestion
no_history_help = "Replace the whole Git history with one new commit."
```

Reply:
Oops, now this triggered a lint warning that the line is too long